modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/CacheMetricsTest.cs

```diff
@@ -118,6 +120,16 @@ public void TestClusterGroupEnableStatistics()
             TestEnableStatistics(
                 cacheName,
                 (cache, b) => cache.Ignite.GetCluster().EnableStatistics(new[] {cacheName}, b));
+
+            var ignite = Ignition.GetIgnite();
+            ignite.CreateCache<int, int>(new CacheConfiguration("clusterEnableStatsValidName"));
+            var cluster = ignite.GetCluster();
+
+            Assert.DoesNotThrow(() => cluster.EnableStatistics(Enumerable.Empty<string>(), true));
+
+            // Nonexistent cache name
+            Assert.Throws<IgniteException>(
```

Comment:
Let's add an assertion for the exception message. Does it make sense, is it clear enough for the user to fix the issue?